float32 height
float32 deviation
float32 min
float32 max
int32 nr_valid_samples